### PR TITLE
Log when diff too large

### PR DESCRIPTION
**Pull request by freva**
Before:
```
[2018-04-24 08:27:52] NodeAdmin: Patching file ip6tables-filter
[2018-04-24 08:27:52] NodeAdmin: Patching file iptables-filter
```
After:
```
[2018-04-24 08:35:50] NodeAgent-host002-10: Patching file ip6tables-filter: Diff too large
[2018-04-24 08:35:50] NodeAgent-host002-10: Patching file iptables-filter: Diff too large
```

### CODE REVIEW

File: node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/Editor.java [hunk 1]
```diff
@@ -102,7 +102,7 @@ public boolean edit(Consumer<String> logConsumer) {
             return false;
         }
 
-        String diffDescription = diffTooLarge(diff) ? "" : ":\n" + diff.toString();
+        String diffDescription = diffTooLarge(diff) ? ": Diff too large" : ":\n" + diff.toString();
```
Comment (hakonhall) [inline node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/Editor.java]: Idea: How about we try to track the size the diff would have had and print that here? Now we have no idea how much larger we need to raise MAX_LENGTH to be able to see the diffs.

File: node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/Editor.java [hunk 2]
```diff
@@ -24,7 +24,7 @@
     private static final Logger logger = Logger.getLogger(Editor.class.getName());
     private static final Charset ENCODING = StandardCharsets.UTF_8;
 
-    private static int maxLength = 300;
+    private static final int MAX_LENGTH = 300;
```
Comment (hakonhall) [inline node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/Editor.java]: From some of the diffs I have seen - I think we can raise this to say 500?

**Review by hakonhall — APPROVED** [review]

File: node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/Editor.java (hunk 2 above)
Comment (freva) [reply node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/Editor.java]: Changed to 500

File: node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/Editor.java (hunk 1 above)
Comment (freva) [reply node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/Editor.java]: Sure, now logs 
```
NodeAgent-host002-10: Patching file ip6tables-filter: Diff too large (518)
```

**Pull request merged**